siteconfig.json5

```diff
@@ -20,10 +23,12 @@
             currentLanguageGroup: "%native",
             genericLanguage: "%native · %local"
         },
-        // The code of the language that would always stay on top of the "Language" setting's menu list.
+        // The code of OS default language `gpii-app` (it might be the one with which the system was installed with).
+        // This language should always stay on top of the "Language" setting's menu list.
         // The code must be exactly matching one of the installed languages' code (with respect to the region).
         // In case the selected language code is missing, there simply won't be a special first setting.
-        defaultLanguage: "en-US"
+        // NOTE: this is not related to the default language value defined for the language setting
+        systemDefaultLanguage: "bg-BG"
```

Comment:
Did this get left behind from testing?

Reply:
Yes, I think that's the case. It should be reverted back to "en-US"